aicsimageio/readers/lif_reader.py

```diff
@@ -0,0 +1,643 @@
+        exceptions.UnsupportedFileFormatError: The file could not be read or is not
+            supported.
+        """
+        with self._fs.open(self._path) as open_resource:
+            lif = LifFile(open_resource)
+            selected_scene = lif.get_image(self.current_scene_index)
+            scene_short_info = selected_scene.info
+
+            # Check for mosaic tiles
+            tile_positions = scene_short_info["mosaic_position"]
+
+            # If there are tiles in the image use mosaic dims
+            if len(tile_positions) > 0:
+                dims = DEFAULT_DIMENSION_ORDER_LIST_WITH_MOSAIC_TILES
+
+            # Otherwise use standard dims
+            else:
+                dims = DEFAULT_DIMENSION_ORDER_LIST
+
+            # Get image data
+            image_data = self._get_image_data(
+                fs=self._fs,
+                path=self._path,
+                scene=self.current_scene_index,
+                retrieve_dims=dims,
+                retrieve_indices=[None] * len(dims),  # Get all planes
+            )
+
+            # Get metadata
+            meta = lif.xml_root
+
+            # Create coordinate planes
+            coords, px_sizes = self._get_coords_and_physical_px_sizes(
+                xml=meta,
+                image_short_info=scene_short_info,
+                scene_index=self.current_scene_index,
+            )
+
+            # Store pixel sizes
+            self._px_sizes = px_sizes
+
+            return xr.DataArray(
+                image_data,
+                dims=dims,
+                coords=coords,  # type: ignore
+                attrs={constants.METADATA_UNPROCESSED: meta},
+            )
+
+    @staticmethod
+    def _stitch_tiles(
+        data: types.ArrayLike,
+        dims: str,
+        ny: int,
+        nx: int,
+    ) -> types.ArrayLike:
+        # Create empty array
+        arr_shape_list = []
+        for dim, size in zip(dims, data.shape):
+            if dim not in [
+                DimensionNames.MosaicTile,
+                DimensionNames.SpatialY,
+                DimensionNames.SpatialX,
+            ]:
+                arr_shape_list.append(size)
+            elif dim == DimensionNames.SpatialY:
+                arr_shape_list.append(size * ny)
+            elif dim == DimensionNames.SpatialX:
+                arr_shape_list.append(size * nx)
+
+        # Fill all tiles
+        rows: List[np.ndarray] = []
+        for row_i in range(ny):
+            row: List[np.ndarray] = []
+            for col_i in range(nx):
+                # Calc m_index
+                m_index = (row_i * nx) + col_i
+
+                # Get tile by getting all data for specific M
+                tile = transforms.reshape_data(
+                    data,
+                    given_dims=dims,
+                    return_dims=dims.replace(DimensionNames.MosaicTile, ""),
+                    M=m_index,
+                )
+
+                # LIF image stitching has a 1 pixel overlap
```

Comment:
is this 1-pixel-overlap generally true for all LIF files?  I thought tiles could be anywhere and have any amount of overlap (I believe we know that is the case for other file types)

Reply:
I couldn't find anything in metadata that specifies what the overlap is for the tiles so I assume its the same for all images. Like if the metadata doesn't have the overlap value then they are all the same right?

Reply:
what I expected was that each tile would know its coordinates in metadata somewhere (like ome-xml planes) and if the coordinates are known then you can position the tiles. Assumption is that the microscope stage can be positioned anywhere in xy space to do a capture.  surprising if that's not the case.

Reply:
Yea so that is how they are returned....

From `readlif` I get back a list of tuples of: `x_index, y_index, x_position, y_position` I just chose to use the indexes instead of the positions.

Reply:
@toloudis does this seem fine?

Reply:
It still seems wrong to me but I suppose we would need a file with different overlaps to test it.

Reply:
Yep. If we get an issue for "my mosaic image looks wrong" we will just have to ask for the image. Sounds good though. Marking resolved and merging